lib/panda.rb

```diff
@@ -18,15 +18,21 @@ def config
       @config ||= Panda::Configuration.new
     end
 
-    def make_get_request(request)
+    def make_request(request)
       connection = Faraday.new do |conn|
         conn.use      Panda::ErrorMiddleware
         conn.request  :json
         conn.response :json
       end
 
-      response = connection.get(request.url, request.params, request.headers)
+      response = do_request(connection, request)
```

Comment:
We can use `connection.run_request(request.method, request.url, request.body, request.headers)`, it's more flexible in case we will need to add more HTTP methods